services/sql-migrator/migrator_test.go

```diff
@@ -42,8 +43,14 @@ func TestMigrate(t *testing.T) {
 				MigrationsTable: fmt.Sprintf("migrations_%s", dir),
 				Handle:          postgre.DB,
 			}
-
-			err := m.Migrate(dir)
+			var err error
+			if strings.HasPrefix(dir, "reports_always") {
+				err = m.MigrateFromTemplates("reports_always", map[string]interface{}{
```

Comment:
can we validate the autovacuum sql by setting the `Reporting.autoVacuumCostLimit` option?

Reply:
Done